stps/sig-network/ip-request-stp.md

```diff
@@ -0,0 +1,216 @@
+| **Topology Considerations**       | [V]   | Evaluated multi-cluster, network topology, and architectural impacts.                                                                                    | Same as in the setups that are used for the MTV IPAM claim testing.<br/>3rd party will provide UDN and annotated VM, CNV testing starts from that.  |
+
+
+### **II. Software Test Plan (STP)**
+
+This STP serves as the **overall roadmap for testing**, detailing the scope, approach, resources, and schedule.
+
+#### **1. Scope of Testing**
+
+Briefly describe what will be tested. The scope must **cover functional and non-functional requirements**. Must ensure user stories are included and aligned to downstream user stories from Section I.
+
+**In Scope:**
+- Testing of known networking flows on VMs, where specific IP is requested for primary interface over
+Cluster/UserDefinedNetwork.\
+The uniqueness in this feature is how the IP is assigned - using the new `network.kubevirt.io/addresses` annotation
+
+**Document Conventions (if applicable):** None
+
+#### **2. Testing Goals**
+
+Define specific, measurable testing objectives for this feature, such as:
+
+- [ ] Goal 1: Verify known networking scenarios are still valid when primary static IP is assigned via the new\
+annotation.
+- [ ] [Goal 2: Consider new cases to be tested, presented by this new feature.
+
+#### **3. Non-Goals (Testing Scope Exclusions)**
+
+Explicitly document what is **out of scope** for testing. **Critical:** All non-goals require explicit stakeholder agreement to prevent "I assumed you were testing that" issues.
+
+**Note:** Replace example rows with your actual non-goals. Each non-goal must have PM/Lead sign-off.
+
+| Non-Goal                                             | Rationale                                                                                                                                           | PM/ Lead Agreement  |
+|:---------------------------------------------------- |:--------------------------------------------------------------------------------------------------------------------------------------------------- |:------------------- |
+| Templating VM spec with the `addresses` annottaion.  | It's the responsibility of the 3rd party integrator, and not accessible by us.<br/>We cannot and will not test the integration with the 3rd party.  | [ ] Name/Date       |
+| Creating C/UDN with IP range.                        | Same                                                                                                                                                | [ ] Name/Date       |
+
+
+**Important Notes:**
+- Non-goals without stakeholder agreement are considered **risks** and must be escalated (see Section II.7 - Risks and Limitations)
+- Review non-goals during Developer Handoff/QE Kickoff meeting (see Section I.2 - Technology and Design Review)
+
+#### **4. Test Strategy**
+
+##### **A. General**
```

Comment:
this section was removed from the template to avoid the duplication with `Test Scenarios & Traceability`

Reply:
Oh, cool. It indeed confused me now that I went over the comments.
Thank you